### PR TITLE
Update Travis clang-format to version 6

### PR DESCRIPTION
**Pull request by DTasev**

### CODE REVIEW

**Comment by SimonHeybrock** [comment]
Can we avoid reformatting the third-party `span` stuff? We should probably exclude it from our `clang-format` job.

**Comment by DTasev** [comment]
I have reverted the changes to `common/`, and added a `.clang-format` file that disables the formatting for that folder

File: common/.clang-format [hunk 1]
```diff
@@ -0,0 +1 @@
+DisableFormat: true
```
Comment (SimonHeybrock) [inline common/.clang-format]: Most of `common` is our own and *should* be formatted, just `common/include/scipp/common/span` is not.

Comment (DTasev) [reply common/.clang-format]: The span folder has it's own `.clang-format` file, so I've just disabled formatting for it there.

**Pull request merged**